.github/workflows/a-team.yml

```diff
@@ -8,6 +8,52 @@ on:
       - main
 
 jobs:
+  aws_setup:
+    if: github.event.pull_request.head.repo.full_name == github.repository
+    env:
+      AWS_ACCESS_KEY_ID: ${{ secrets.ATEAM_AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.ATEAM_AWS_SECRET_ACCESS_KEY }}
+      TF_VAR_ORG: osbuild
+      TV_VAR_REPO: automotive-ci
+      GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
```

Comment:
Can you please rename the secret to align with the variable?

Reply:
Or is it something GitHub pre-populates?

Reply:
I can and I can't the `GITHUB_` prefix is a protected prefix and GH will not allow us to prefix secrets with this, the env var on the other hand needs to be `GITHUB_TOKEN` in order for it to be recognised by the github terraform module

Reply:
+1